Docs, docs, docs. (#576)
mozilla/code-review: docs/configuration.md

# Note: The redis connection string is provided by Heroku as an environment variable
events:
  # Sentry connection string to publish system exceptions
  # Each project has its own Sentry environment
  SENTRY_DSN: https://xxx:yyy@sentry.com

  # Does the system need to listen for autoland payloads on pulse
  # This is needed to ingest autoland issues in the backend
  autoland_enabled: true

  # Pulse authentication to get messages for the autoland triggers
  pulse_user: xxx
  pulse_password: yyy

  # Parameters for the test selection feature
  # TODO: Marco may document a bit more here
  bugbug_phabricator_deployment: dev
  test_selection_enabled: false
  test_selection_share: 0.03
  test_selection_notify_addresses:
    - admin@mozilla.com

  # Skip all revisions produced by these users
  # It's useful to avoid huge revisions produced by bots
  user_blacklist:
    - bot-username-a
    - bot-username-b

  # Taskcluster credentials for the community instance
  # Needed to run the bugbug analysis
  taskcluster_community:
    client_id: project/relman/bugbug/xxxx
    access_token: yyy

  # Pulse authentication on the community Taskcluster instance

>   # to get messages for the risk analysis triggers

Reviewer: ```suggestion
  # to get messages for the test selection triggers
```

  communitytc_pulse_user: xxxx
  communitytc_pulse_password: yyy

>   # Phabricator usernames for whom a risk analysis will be built

Reviewer: ```suggestion
  # Phabricator usernames for whom a risk analysis will be triggered
```

>   # when they are requested as reviewers on a patch

Reviewer: ```suggestion
  # when they are authors or reviewers of a patch
```

  risk_analysis_users:
    - user-a
    - user-b

  # The SSH private key used to push patches on the Try server

>   # It can be setup globally for the events project, or specified per repositories

Reviewer: ```suggestion
  # It can be setup globally for the events project, or specified per repository
```
